stylix/palette.nix

```
@@ -10,7 +10,7 @@ let
 
   paletteJSON = let
     generatedJSON = pkgs.runCommand "palette.json" { } ''
-      ${palette-generator}/bin/palette-generator ${cfg.polarity} ${cfg.image} $out
+      ${palette-generator}/bin/palette-generator ${cfg.polarity} ${lib.escapeShellArg cfg.image} $out
```

What about simply quoting the argument in Bash:

```suggestion
      ${palette-generator}/bin/palette-generator ${cfg.polarity} "${cfg.image}" $out
```

Additionally, we should quote the other arguments:

```suggestion
      ${palette-generator}/bin/palette-generator "${cfg.polarity}" "${cfg.image}" "$out"
```

Personally, I would wrap the code at 80 characters:

```suggestion
      ${palette-generator}/bin/palette-generator \
        "${cfg.polarity}" \
        "${cfg.image}" \
        "$out"
```

---

`lib.escapeShellArg` is quoting the argument, plus escaping any quotes within it.

https://github.com/NixOS/nixpkgs/blob/2351f04cd4719fd8c1e121e1280757da66bdd2fb/lib/strings.nix#L442

I'll post a comment in #318 regarding further changes.